include/seqan3/io/stream/debug_stream.hpp

```diff
@@ -295,6 +296,34 @@ inline debug_stream_type & operator<<(debug_stream_type & s, alphabet_t const l)
     return s << to_char(l);
 }
 
+namespace detail
+{
+    //!\brief Helper function to print elements of a tuple separately.
+    template<typename tuple_t, std::size_t ...I>
+    void print_tuple(debug_stream_type & s, tuple_t const & t, std::index_sequence<I...> const &)
+    {
+        ((s << (I == 0 ? "" : ",") << std::get<I>(t)), ...);
+    }
+} // namespace detail
+
+/*!\brief All tuples can be printed by printing their elements separately.
+ * \tparam tuple_t Type of the tuple to be printed; must model seqan3::tuple_like_concept.
+ * \param s The seqan3::debug_stream.
+ * \param t The tuple.
+ * \relates seqan3::debug_stream_type
```

Comment:
`\return` statement missing

Reply:
The other overloads also don't specify a `\return` statement.
Anyway, it would return a reference to itself?

Reply:
fine